src/app/(sok)/_utils/elasticSearchRequestBody.js

```diff
@@ -604,32 +603,15 @@ function mainQueryConjunctionTuning(qAsArray) {
         "adtext_no^0.2",
         "employerdescription_no^0.1",
     ];
-    const q = qAsArray.join(" ");
+    const q = qAsArray.join(" ").trim();
```

Comment:
brukes denne?

Reply:
Ja, i ID-matchen. Flyttar in den dit så, blir det tydligare.